app-backend/database/src/main/scala/com/azavea/rf/database/tables/ScenesToProjects.scala

```diff
@@ -111,8 +111,31 @@ object ScenesToProjects extends TableQuery(tag => new ScenesToProjects(tag)) wit
       )
     }
   }
+  
+  def setColorCorrectParamsBatch(projectId: UUID, params: BatchParams)(implicit database: DB) = {
+    val results = for {
```

Comment:
We should do this as a single transaction in order to avoid creating a partially applied batch update. As it is, the request will error out, but still apply as many updates as it can, possibly resulting in a project where half of the scenes have an updated color correction the next time the project is loaded from the frontend